### PR TITLE
Added More Resources in resources-list.md

### PR DESCRIPTION
**Pull request by ronika-das**
I have added:
1. Hackathons
2. Scholarships
3. Some more open source programs

I have written Conferences and Hackathons separately here. pls let me know any suggestions

### CODE REVIEW

File: women-in-tech-resources/resources-list.md [hunk 1]
```diff
@@ -76,30 +81,34 @@ If this is your first time looking at the Women in Tech Resources List, spend 30
...
+- [Linkedin Wintathon](https://wintathon2019.splashthat.com/)
+- [HackInOut](https://hackinout.co/)
+- [Smart India Hackathon](https://www.sih.gov.in/)
+
 ### Open Source Programs
 - [Google Summer of Code - Global](https://summerofcode.withgoogle.com/)
 - [GirlScript Summer Of Code](https://gssoc.tech/)
 - [Outreachy](https://www.outreachy.org/)
 - [Rails Girls Summer of Code](https://railsgirlssummerofcode.org/)
+- [Hacktober](https://hacktoberfest.digitalocean.com/)
```
Comment (manaswinidas) [inline women-in-tech-resources/resources-list.md]: Hacktoberfest in place of Hacktober

Comment (ronika-das) [reply women-in-tech-resources/resources-list.md]: done

File: women-in-tech-resources/resources-list.md [hunk 2]
```diff
@@ -76,30 +81,34 @@ If this is your first time looking at the Women in Tech Resources List, spend 30
...
+- [Linkedin Wintathon](https://wintathon2019.splashthat.com/)
+- [HackInOut](https://hackinout.co/)
+- [Smart India Hackathon](https://www.sih.gov.in/)
+
 ### Open Source Programs
 - [Google Summer of Code - Global](https://summerofcode.withgoogle.com/)
 - [GirlScript Summer Of Code](https://gssoc.tech/)
 - [Outreachy](https://www.outreachy.org/)
 - [Rails Girls Summer of Code](https://railsgirlssummerofcode.org/)
+- [Hacktoberfest](https://hacktoberfest.digitalocean.com/)
```
Comment (alodahl) [inline women-in-tech-resources/resources-list.md]: Could you put hacktoberfest under the Bay Area topic too? We were part of that last year!

**Comment by alodahl** [comment]
It’s ready to merge! Please change this from a draft to a regular PR and I will merge 😁

**Pull request merged**